engine/flink/management/src/main/scala/pl/touk/nussknacker/engine/management/FlinkConfig.scala

```diff
@@ -15,12 +18,14 @@ import scala.concurrent.duration.{DurationInt, FiniteDuration}
 final case class FlinkConfig(
     restUrl: Option[String],
     jobManagerTimeout: FiniteDuration = 1 minute,
+    // TODO: move to scenarioTesting
```

Comment:
leaving this todo?

Reply:
Yep, I don't think that it is worth to refactor it now